config/config.js.sample

```diff
@@ -21,6 +21,10 @@ var config = {
 	                                                       // or IPv4 range of 192.168.3.0 --> 192.168.3.15 use CIDR format :
 	                                                       // ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.3.0/28"],
 
+	useHttps: false,			// Support HTTPS or not
+	httpsPrivateKey: "",		// HTTPS Private Key path
```

Comment:
I think it’s a good idea to explain that these variables are optional and only necessary if you want to use https. Agree?

Reply:
Yeah.... I will update it with below remark. Is that fine ? 

useHttps: false,		// Support HTTPS or not, default "false" will use HTTP
httpsPrivateKey: "",		// HTTPS private key path, only require when useHttps is true
httpsCertificate: "",		// HTTPS Certificate path, only require when useHttps is true